impl/src/main/java/com/okta/authn/sdk/impl/resource/DefaultAuthenticationResponse.java

```diff
@@ -199,7 +202,7 @@ public User getUser() {
 
     @Override
     public Integer getCorrectAnswer() {
-        Map<String, Object> rawFactor = (Map<String, Object>) getEmbedded().get(NESTED__FACTOR_PROPERTY.getName());
+        Map<String, Object> rawFactor = (Map<String, Object>) getEmbedded().get(NESTED__FACTORS_PROPERTY.getName());
```

Comment:
@arvindkrishnakumar-okta Thank you for adding this change in the latest OKTA release. I was implementing this change on our end and noticed that the `key` while fetching the `rawFactor` is incorrect. Instead of `NESTED__FACTORS_PROPERTY` (factors) it should be `NESTED__FACTOR_PROPERTY` (factor). I have attached a screenshot of the response for your reference. If needed I can make the change and create a new PR if you think this is correct.

<img width="762" alt="Screen Shot 2021-03-01 at 3 42 08 PM" src="https://user-images.githubusercontent.com/10443157/109557027-8e44d400-7aa5-11eb-8874-43644ffa288b.png">

Reply:
Thanks for noticing this! Will fix it in #98.

Reply:
Thank you 👍